jiant/tasks/lm.py

```diff
@@ -172,3 +179,132 @@ def __init__(self, path, *args, **kw):
             "val": os.path.join(path, "valid.sentences.txt"),
             "test": os.path.join(path, "test.sentences.txt"),
         }
+
+
+@register_task("wikipedia_corpus_mlm", rel_path="wikipedia_corpus_small/")
+class MaskedLanguageModelingTask(Task):
+    """
+    Masked language modeling task on Wikipedia dataset
+    Attributes:
+        max_seq_len: (int) maximum sequence length
+        min_seq_len: (int) minimum sequence length
+        files_by_split: (dict) files for three data split (train, val, test)
+    We are currently using an unpreprocessed version of the Wikipedia corpus
```

Comment:
@phu-pmh  can you double check this description of what dataset we're currently using?

Reply:
We did hyperlink and xml removal etc, and did sentence-level tokenization. Each sentence is not processed.